consult-dir.el

```diff
@@ -119,10 +140,43 @@ arguments and return a list of directories."
   '(consult-dir--source-bookmark
     consult-dir--source-default
     consult-dir--source-project         ;projectile if available, project.el otherwise
-    consult-dir--source-recentf)
+    consult-dir--source-recentf
+    consult-dir--source-tramp-local
```

Comment:
Do not include the tramp sources by default, except for `consult-dir--source-tramp-local`.